src/Javis.jl

```diff
@@ -262,6 +265,14 @@ function render(
         @error "Currently, only gif and mp4 creation is supported. Not a $ext."
     end
 
+    if livestream == true
```

Comment:
could be `if livestream` actually we should do that for `if liveview` above too :smile: